core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/docker/DockerClient.scala

```diff
@@ -204,7 +205,12 @@ class DockerClient(dockerHost: Option[String] = None,
       s"running ${cmd.mkString(" ")} (timeout: $timeout)",
       logLevel = InfoLevel)
     executeProcess(cmd, timeout).andThen {
-      case Success(_) => transid.finished(this, start)
+      case Success(_) =>
+        MetricEmitter.emitHistogramMetric(
+          LogMarkerToken("docker", "runCmd", "duration", Some(args.head), Map("cmd" -> args.head))(
+            MeasurementUnit.time.milliseconds),
+          Instant.now.toEpochMilli - transid.meta.start.toEpochMilli)
```

Comment:
This is not the usual way to send metric.
Is there any reason not to use transid start, finish ?

Reply:
Already changed to use `transid.finish` to send the metric.